composer.json

```diff
@@ -22,10 +22,13 @@
     ],
     "homepage": "https://codeception.com/",
     "require": {
-        "php": "^8.0",
+        "php": "^8.1",
         "codeception/codeception": "*@dev",
         "codeception/lib-asserts": "^2.0"
     },
+    "require-dev": {
+        "behat/gherkin": "~4.11.0"
```

Comment:
I had to pin the version because of https://github.com/Codeception/Codeception/issues/6833

Reply:
https://github.com/Codeception/Codeception/pull/6836 should make this workaround obsolete.